scripts/modal.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function installModal () {
+  // Get the modal
+  const modal = document.getElementById('myModal')
+
+  // Get the <span> element that closes the modal
+  const span = document.getElementsByClassName('close')[0]
+
+  // When the user types ?, open the modal
+  window.addEventListener('keydown', event => {
+    if (event.key === '?') {
+      if (event.target === null || event.target.tagName !== 'INPUT') {
+        modal.style.display = 'block'
+        event.stopImmediatePropagation()
+        event.preventDefault()
+      }
+    }
+
+    if (event.keyCode === 27) {
```

Comment:
How would you feel about including `?` in this, so it could be toggled?

```suggestion
    if (event.keyCode === 27 || event.key === '?') {
```